### ISSUE TITLE
create package task removes custom folders

### ISSUE THREAD

**Issue by jhercher** [ExLibrisGroup/primo-explore-devenv#8]
Hi,

i placed a subfolder named "fonts" in `/primo-explore/custom/[View]`
The GULP `createPackage`-tasks removes it from the zip created in `/packages/[View].zip]`
Can you please give me a hint how to prevent this, I'm not that experienced with gulp yet.

Thanks

**Comment by noamamit92** [comment]
The concept of the customization process is to have 2 tools:
1. The package - a well structured zip file - with constant structure that can be edited in various ways and uploaded to the BO
2. The development environment - A tool to help you create the package - using whatever tools you, as a developer, need.
This means that you can split your code to multiple sources as you did or use advanced technologies such as saas and babel   - just make sure your compiled code fits the above package structure.

To that cause I created the create-package script - it will take the package based files and make sure only those files are loaded to the BO

So basically my recommendation is : 
Manage your source code in GitHub, publish your compiled code to the BO - only the run time files.

What is the content of your /fonts directory?

**Comment by jhercher** [comment]
The /fonts directory contains files for webfonts, e.g. to easily create icons using css-classes. See [fontawesome.io](http://fontawesome.io). Basically that files ending with .eot|.svg|.ttf|.woff|.woff2|.otf

Since these files are removed i need to unpack the zip insert the folder and zip again before uploading to the BO...

**Comment by noamamit92** [comment]
If you put these files directly in the img directory - they should get added to the zip

**Comment by noamamit92** [comment]
@jhercher - did you try the suggested solution?
